src/app/lists/ListDataview.tsx

```diff
@@ -74,6 +101,23 @@ const ListsDataView: React.FC<Props> = (props) => {
         throw new Error("No data found");
     }
 
+    const rows = props.data.map((row) => {
+        const newRow: ListRow = {
+            itemName: row.item_name,
+            "1": { quantity: row["1_quantity"], notes: row["1_notes"] },
+            "2": { quantity: row["2_quantity"], notes: row["2_notes"] },
+            "3": { quantity: row["3_quantity"], notes: row["3_notes"] },
+            "4": { quantity: row["4_quantity"], notes: row["4_notes"] },
+            "5": { quantity: row["5_quantity"], notes: row["5_notes"] },
+            "6": { quantity: row["6_quantity"], notes: row["6_notes"] },
+            "7": { quantity: row["7_quantity"], notes: row["7_notes"] },
+            "8": { quantity: row["8_quantity"], notes: row["8_notes"] },
+            "9": { quantity: row["9_quantity"], notes: row["9_notes"] },
+            "10": { quantity: row["10_quantity"], notes: row["10_notes"] },
```

Comment:
Same here, is it `10+_quantity`

Reply:
no, these are the keys on the schema and there it's called `10_quantity` and `10_notes`